go/service/delegate_ui.go

```diff
@@ -47,3 +47,14 @@ func (d *DelegateUICtlHandler) RegisterUpdateUI(_ context.Context) error {
 	d.G().UIRouter.SetUI(d.id, libkb.UpdateUIKind)
 	return nil
 }
+
+func (d *DelegateUICtlHandler) RegisterRekeyUI(_ context.Context) error {
+	d.G().UIRouter.SetUI(d.id, libkb.RekeyUIKind)
+
+	// Let Gregor related code know that a RekeyUI client
```

Comment:
I'm not sure I understand your question.  This is what electron calls to register itself as a RekeyUI provider.

The comment is copied from RegisterIdentifyUI.